Sources/ArgumentParser/Parsable Types/AsyncParsableCommand.swift

```diff
@@ -31,6 +31,10 @@ extension AsyncParsableCommand {
   /// Instead of calling this method directly, you can add `@main` to the root
   /// command for your command-line tool.
   public static func main() async {
+#if DEBUG
+    configuration.subcommands.forEach(checkAsycHierarchy)
+#endif
+    
```

Comment:
We don't need these checks in the async main() methods — if one of these is executing, the subcommands can be any mix of synchronous or asynchronous commands.

Reply:
I want to confirm, in the following hierarchy structure, will the innermost `AsyncCommand` also be executed asynchronously?
```
AsyncCommand { Command { AsyncCommand } }
```